services/auths.js

```diff
@@ -23,3 +24,23 @@ exports.verifyEmail = async id => {
     return error.message;
   }
 };
+
+exports.loginUser = async (userData) => {
+  const { email, password, username } = userData;
+
+  try {
+    const user = await getBy({email});
+    if(user && bcrypt.compareSync(password, user.password)) {
+      const token = generateToken(user);
```

Comment:
You need an auth middleware that issues a token as well as a refresh token for when the old one expires. Look into it.